tests/cpp/tests/test_nodes_mathematical.cpp

```diff
@@ -404,32 +404,7 @@ TEMPLATE_TEST_CASE("NaryOpNode", "", functional::max<double>, functional::min<do
     }
 
     GIVEN("A vector<Node*> of constants") {
-        auto nodes = std::vector<Node*>{
```

Comment:
Why is this removed?

Reply:
Oh I think I see, this constructor for `NaryOpNode` was removed, and the below tests covers the behavior.